lib/fragmentSender.js

```diff
@@ -210,6 +231,9 @@ export default async (chat, fragments, payload, initialMessage = '', initialMedi
                                 report: payload.report,
                                 type: 'report',
                                 preview: payload.preview,
+                                category: payload.category,
+                                event: payload.event,
+                                label: `quiz-${n}`,
```

Comment:
wouldn't it be nicer to have `option.quiz_option` as the label instead of some number?